disnake/embeds.py

```diff
@@ -305,8 +310,7 @@ def __bool__(self) -> bool:
                 self.title,
                 self.url,
                 self.description,
-                # not checking for falsy value as `0` is a valid color
-                self._colour not in (MISSING, None),
+                self._colour,
```

Comment:
what is this change for?

Reply:
`_colour` isn't an `int` but a `Colour` instance, so there's no need for handling this in a special way; `bool(Colour(0))` is `True`. Functionally this doesn't make a difference.